pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.ruff]
+# Line length (same as Black)
+line-length = 88
+
+# Python target version
+target-version = "py38"
+
+# Ignored rules for the entire project
+[tool.ruff.lint]
+ignore = [
+    "E501",   # Line too long
+    "E203",   # Whitespace before ':'
+    "TRY301", # Raise within try block
+    # "W503"  # Line break before binary operator (not PEP8 enforced, so not implemented in Ruff)
+]
+
+# flake8 plugins to enable:
+# - flake8-bugbear          B
+# - flake8-builtins         A
+# - flake8-comprehensions   C4
+# - flake8-debugger         T10
+# - flake8-logging-format   G
```

Comment:
I am also not the most fit to review this, but I do have a question. Why is `G` not included in the list below?

Reply:
Because I simply forgot it! 😅 I did it manually, no Copilot. Good catch! 🔎